usr/sbin/pkcscca/pkcscca.c

```diff
@@ -792,7 +792,7 @@ int add_key(CK_OBJECT_HANDLE handle, CK_ATTRIBUTE *attrs, struct key **keys)
 }
 
 int find_wrapped_keys(CK_FUNCTION_LIST *funcs, CK_SESSION_HANDLE sess,
-                      CK_KEY_TYPE *key_type, struct key **keys, CK_BBOOL all)
+                      CK_KEY_TYPE *key_type, struct key **keys)
 {
```

Comment:
In the first commit you introduced the CK_BBOOL all parm, and now you are removing it?
Also: in the first commit you added a CKA_CLASS to the key_tmpl array, but now you remove it?